mlflow/server/js/src/common/components/ErrorView.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import errorDefaultImg from '../static/default-error.svg';
+import error404Img from '../static/404-overflow.svg';
+import Colors from '../../experiment-tracking/styles/Colors';
+import Routes from '../../experiment-tracking/routes';
+import { Link } from 'react-router-dom';
+
+const altMessages = {
+  400: '400 Bad Request',
+  401: '401 Unauthorized',
+  404: '404 Not Found',
+  409: '409 Conflict',
+  500: '500 Internal Server Error',
+  502: '502 Bad Gateway',
+  503: '503 Service Unavailable',
+};
+
+function ErrorImage(props) {
```

Comment:
Let's add some basic rendering tests for this new component.

Reply:
Some tests were added